test/In.ProjectEKA.HipServiceTest/DataFlow/Builder/TestBuilder.cs

```diff
@@ -0,0 +1,23 @@
+using In.ProjectEKA.HipService.DataFlow;
+
+namespace In.ProjectEKA.HipServiceTest.DataFlow.Builder
+{
+    using Bogus;
+
+    public static class TestBuilder
+    {
+        private static Faker faker;
+        
+        internal static Faker Faker() => faker ??= new Faker();
+        
+        internal static HealthInformationRequest HealthInformationRequest(string transactionId)
```

Comment:
Can't we use Faker<HealthInformationRequest>?

Reply:
tried using it but it comes as null